Automattic/wp-calypso
# Plugin Details: Update the sidebar design

#### Proposed Changes

* Update the sidebar min-width to `300px`
* Add the needs plan upgrade message
* Update the USPS styles
* Handle the edge-cases for the CTA

#### Out of scope 
- Manage sites - #63888
- Activate/Deactivate, etc. - #65611

#### Testing Instructions
* Enable the feature flag `plugins/plugin-details-layout`*

---

* Go to a plugin page without a selected site. Ex: `plugins/wordpress-seo`
* You should see the `Manage sites` button

---

* Go to a plugin installed on the selected site. Ex: `plugins/wordpress-seo/{site}`
* You should see the install and active (or inactive) text and the activate/deactivate button.


---

* Go to a free plugin not installed on your business/ecommerce/starter/pro site. Ex: `/plugins/google-site-kit/:site:`
* You should see the `Install and activate` button.

---

* Go to a free plugin not installed on your free site. Ex: `/plugins/google-site-kit/:site:`
*  You should see the `Install and activate` button and the `Upgrade needed` message.

---

* Go to a WooCommerce paid plugin from your business/ecommerce/starter/pro site. Ex: `/plugins/woocommerce-subscriptions/{site}`
* You should see the purchase buttons along with all the available USPS (store foundation, support, etc)

---

* Go to a WooCommerce paid plugin from your free site. Ex: `/plugins/woocommerce-subscriptions/{site}`
* You should the purchase and plans USPS along with the other USPS

---

* Disable the `plugins/plugin-details-layout` feature flag and go to plugins pages.
* The result should be the same as available on WordPress.com

---

\* To enable this feature flag add `?flags=plugins/plugin-details-layout` to the end of your URL or past the following code on the developer tools console `document.cookie = 'flags=plugins/plugin-details-layout;max-age=1209600;path=/'`

---

Fixes #63714
